Tutorials/octave/octave_demo.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm.octave import *
+from qm.qua import *
+from configuration import *
+
+opx_ip = "xxx.xxx.xxx.xxx"
+opx_port = 80
+octave_ip = "xxx.xxx.xxx.xxx"
+octave_port = 80
```

Comment:
```suggestion
qop_ip = "xxx.xxx.xxx.xxx"
opx_port = 80
octave_port = 80
```
We set up the devices to have the same IP but different ports. 
Of course, these changes also need to be reflected down.

Reply:
OK, I wanted it to be as general as possible in that sense